sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -83,4 +90,63 @@ public void testBatch() throws Exception {
             assertThat(dao.findNameById(ids[1])).isEqualTo("Macklin");
         });
     }
+
+    @Test
+    public void testUpdateMultipleGeneratedKeys() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
+            handle.execute("create table something (id serial, name varchar(50), created_on timestamp default now())");
+
+            GetMultipleGeneratedKeysDao dao = handle.attach(GetMultipleGeneratedKeysDao.class);
+
+            IdCreateTime result = dao.insert("foo");
+            assertThat(result.id).isEqualTo(1);
+            assertThat(result.createdOn).isNotNull();
+        });
+    }
+
+    @Test
+    public void testBatchMultipleGeneratedKeys() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
+            handle.execute("create table something (id serial, name varchar(50), created_on timestamp default now())");
+
+            GetMultipleGeneratedKeysDao dao = handle.attach(GetMultipleGeneratedKeysDao.class);
+
+            List<IdCreateTime> results = dao.insertBatch("foo", "bar");
+            assertThat(results).extracting(ic -> ic.id).containsExactly(1L, 2L);
+            assertThat(results).extracting(ic -> ic.createdOn).hasSize(2).doesNotContainNull();
+        });
+    }
+
+    public interface GetMultipleGeneratedKeysDao {
+        @SqlUpdate("insert into something(name) values (:name)")
+        @GetGeneratedKeys
+        @UseRowMapper(IdCreateTimeMapper.class)
+        IdCreateTime insert(String name);
+
+        @SqlBatch("insert into something(name) values (:name)")
+        @GetGeneratedKeys
+        @UseRowMapper(IdCreateTimeMapper.class)
+        List<IdCreateTime> insertBatch(String... name);
+    }
```

Comment:
Probably should add a separate test that registers a mapper instead of `@UseRowMapper`, just to be thorough.